src/gui/plugins/plotting/Plotting.qml

```diff
@@ -0,0 +1,5 @@
+import "qrc:/qml"
```

Comment:
Missing copyright header.

Also, it would be nice to add minimum layout sizes.